integration-tests/src/test/java/oracle/kubernetes/operator/ITMultipleClusters.java

```diff
@@ -159,58 +176,91 @@ public void testCreateDomainTwoMixedCluster() throws Exception {
-    if (!Files.exists(Paths.get(template + ".org"))) {
-      Files.copy(Paths.get(template), Paths.get(template + ".org"));
+  @Test
+  public void testCreateDomainTwoClusterWDTInImage() throws Exception {
+    String DOMAINUID = "twoclusterdomainwdt";
+    String testMethodName = new Object() {}.getClass().getEnclosingMethod().getName();
+    logTestBegin(testMethodName);
+    logger.info("Creating Operator & waiting for the script to complete execution");
+    if (operator1 == null) {
+      operator1 = TestUtils.createOperator(OPERATOR1_YAML);
     }
-    Files.write(Paths.get(template), add.getBytes(), StandardOpenOption.APPEND);
-    byte[] readAllBytes = Files.readAllBytes(Paths.get(template));
-    logger.info(new String(readAllBytes, StandardCharsets.UTF_8));
+    Domain domain = null;
+    boolean testCompletedSuccessfully = false;
+    try {
+      Map<String, Object> domainMap = TestUtils.loadYaml(DOMAININIMAGE_WDT_YAML);
+      domainMap.put("domainUID", DOMAINUID);
+      if ((System.getenv("LB_TYPE") != null && System.getenv("LB_TYPE").equalsIgnoreCase("VOYAGER"))
+          || (domainMap.containsKey("loadBalancer")
+              && ((String) domainMap.get("loadBalancer")).equalsIgnoreCase("VOYAGER"))) {
+        domainMap.put("voyagerWebPort", new Integer("30377"));
+      }
+      domainMap.put(
+          "customWdtTemplate",
+          BaseTest.getProjectRoot()
+              + "/integration-tests/src/test/resources/multipleclusters/wdtmultipledynclusters.yml");
+      addCluster2ToDomainTemplate(domainMap);
+      domain = TestUtils.createDomain(domainMap);
+      domain.verifyDomainCreated();
+      String pods[] = {
+        DOMAINUID + "-" + domain.getAdminServerName(),
+        DOMAINUID + "-managed-server1",
+        DOMAINUID + "-managed-server2",
+        DOMAINUID + "-managed-server-21",
+        DOMAINUID + "-managed-server-22",
+      };
+      verifyServersStatus(domain, pods);
+      testBasicUseCases(domain);
+      if (!SMOKETEST) {
+        domain.testWlsLivenessProbe();
+      }
+      testCompletedSuccessfully = true;
+    } finally {
+      if (domain != null && !SMOKETEST && (JENKINS || testCompletedSuccessfully)) {
+        domain.destroy();
+      }
+    }
+    logger.log(Level.INFO, "SUCCESS - {0}", testMethodName);
   }
 
   /**
-   * Restore the domain template to original state when test is finished
+   * Append a second cluster to the domain template
    *
-   * @throws IOException
+   * @throws IOException when append fails
    */
-  private void restoreDomainTemplate() throws IOException {
+  private void addCluster2ToDomainTemplate(Map<String, Object> domainMap) throws IOException {
+    String add =
+        "  - clusterName: %CLUSTER_NAME%-2\n"
+            + "    serverStartState: \"RUNNING\"\n"
+            + "    replicas: %INITIAL_MANAGED_SERVER_REPLICAS%\n";
+    String customDomainTemplate =
+        System.getProperty("java.io.tmpdir") + "/customDomainTemplate.yaml";
```

Comment:
use BaseTest.getResultDir() for temp files

Reply:
done